src/doc/trpl/inline-assembly.md

```diff
@@ -100,6 +100,21 @@ fn main() {
 }
 ```
 
+If you would like to use real operands in this position, however,
+you are required to put curly braces `{}` around the register that
+you want, and you are required to put the specific size of the
```

Comment:
Oh, or maybe just `{a}`? It seems that `{ax}` always works (I see no difference in the asm between `al` and `ax`), but `a` presumably means something different to LLVM.